draft-ietf-scitt-architecture.md

```diff
@@ -241,6 +246,10 @@ Issuer:
 : organizations, stakeholders, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers.
 An Issuer may be the owner or author of Artifacts, or an independent third party such as an auditor, reviewer or an endorser.
 
+Non-equivocation:
+
+: a state where Equivocation cannot or does not exist.
```

Comment:
```suggestion
: assurance an Issuer is unable to make different Statements to different Verifiers.
The Transparency Service registers Signed Statements from Issuers, returning consistent results to all Verifiers. {{NON-EQUIVOCATION}}
Over time, an Issuer may register new Signed Statements about an Artifact reflecting new information, however, the collection of statements about the Artifact are consistently returned to all Verifiers.
```

Reply:
252-253 these would be much better if re-cast in terms of tamper evidence.

"The Transparency Service registers Signed Statements maintains statements from Issuers in a manner that enables all verifiers to assure a consistent history. {{NON-EQUIVOCATION}}"

"Over time, an Issuer may register new Signed Statements about an Artifact reflecting new information, however, the consistency of a collection of statements about the Artifact can be checked by all Verifiers"

Reply:
I used some of this in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/131/commits/45f3f12715782df36a37709da07ec19e7db2b9fe, thanks!